src/lib/transaction_protocol_state/transaction_protocol_state.mli

```diff
@@ -5,7 +5,7 @@ module Block_data : sig
   module Stable : sig
     module V2 : sig
       type t = Mina_state.Protocol_state.Body.Value.Stable.V2.t
-      [@@deriving sexp]
```

Comment:
You didn't need to add `yojson` for all of these. You could just have used `sexp`.

Reply:
Small changes like these to the MIna repository make us more out of sync, and without explicit notes, it's hard to remember it's been changed. Ideally you would be working to upstream small changes like these, but in this case it's not actually necessary AFAICT.